tensorflow/core/kernels/transpose_op.cc

```diff
@@ -91,6 +91,18 @@ REGISTER_KERNEL_BUILDER(Name("InvertPermutation")
                             .HostMemory("y"),
                         InvertPermutationOp<int64>);
 
+REGISTER_KERNEL_BUILDER(Name("InvertPermutation")
+                            .Device(DEVICE_DEFAULT)
```

Comment:
Can we just replace the registration for `DEVICE_GPU` with `DEVICE_DEFAULT`? The `DEVICE_DEFAULT` rule would work for `DEVICE_GPU` too. Same for other PRs.

Reply:
@penpornk Hi, I've changed all of my `DEVICE_DEFAULT` PRs. Please review.